src/components/product-filters/product-filters.hbs

```diff
@@ -13,54 +13,67 @@
+                <input id="blue" data-js="filter" class="checkbox__input" aria-label="Filter on color blue" type="checkbox" name="color" value="blue">
+                <label for="blue" class="checkbox__label">Blue</label><br>
+                <input id="green" data-js="filter" class="checkbox__input" aria-label="Filter on color green" type="checkbox" name="color" value="green">
+                <label for="green" class="checkbox__label">Green</label><br>
+                <input id="yellow" data-js="filter" class="checkbox__input" aria-label="Filter on color yellow" type="checkbox" name="color" value="yellow">
+                <label for="yellow" class="checkbox__label">Yellow</label><br>
+                <input id="red" data-js="filter" class="checkbox__input" aria-label="Filter on color red" type="checkbox" name="color" value="red">
+                <label for="red" class="checkbox__label">Red</label>
+            </fieldset>
+            <a class="skip-link" href="#skip-to-products-from-filters">Skip to Products</a>
+        </li>
+        <li class="accordion-panel filter">
+            <button class="accordion-panel__heading filter__heading" type="button">
+                <h3 class="form--label">Size</h3>
+            </button>
+            <fieldset class="accordion-panel__content filter__options">
+                <legend tabindex="0" id="sizeLegend" class="visibly-hidden">Filter by Size</legend>
+                <input id="large" data-js="filter" class="checkbox__input" aria-label="Filter on size large" type="checkbox" name="size" value="large" />
+                <label for="large" class="checkbox__label">Large</label><br>
+                <input id="medium" data-js="filter" class="checkbox__input" aria-label="Filter on size medium" type="checkbox" name="size" value="medium">
+                <label for="medium" class="checkbox__label">Medium</label><br>
+                <input id="small" data-js="filter" class="checkbox__input" aria-label="Filter on size small" type="checkbox" name="size" value="small">
+                <label for="small" class="checkbox__label">Small</label><br>
+            </fieldset>
+            <a class="skip-link" href="#skip-to-products-from-filters">Skip to Products</a>
+        </li>
+        <li class="accordion-panel filter">
+            <button class="accordion-panel__heading filter__heading" type="button">
+                <h3 class="form--label">Price Range</h3>
+            </button>
+            <form data-js="price-filter" class="price-filter__form accordion-panel__content filter__options">
+                <input type="hidden" name="priceFilterFrom" id="priceFilterFrom" value="">
+                <input type="hidden" name="priceFilterTo" id="priceFilterTo" value="">
+                <fieldset class="price-filter__fieldset">
+                    <legend tabindex="0" id="priceLegend" class="visibly-hidden">Filter by Price</legend>
```

Comment:
Text such as this shouldn't be focusable, especially when it is visibly hidden.

Reply:
Ok, I'll fix this

- [ ] Remove `tabindex=0` from the `<legend>` tags

Reply:
@brianelton They're removed